includes/functions.php

```diff
@@ -186,7 +186,7 @@ function dokan_redirect_if_not_seller( $redirect = '' ) {
  *
  * @return array
  */
-function dokan_count_posts( $post_type, $user_id ) {
+function dokan_count_posts( $post_type, $user_id, $exclude_product_type = 'booking' ) {
```

Comment:
make exclude_product_type to array and change code accordingly. in the future, we might need to exclude auction products also.

Reply:
Code changed as you instructed.